stdlib/importlib/abc.pyi

```diff
@@ -70,17 +77,17 @@ class MetaPathFinder(Finder):
 
 class PathEntryFinder(Finder):
     def find_module(self, fullname: str) -> Loader | None: ...
-    def find_loader(self, fullname: str) -> tuple[Loader | None, Sequence[_Path]]: ...
+    def find_loader(self, fullname: str) -> tuple[Loader | None, Sequence[str]]: ...
     def invalidate_caches(self) -> None: ...
     # Not defined on the actual class, but expected to exist.
     def find_spec(self, fullname: str, target: types.ModuleType | None = ...) -> ModuleSpec | None: ...
 
 class FileLoader(ResourceLoader, ExecutionLoader, metaclass=ABCMeta):
     name: str
-    path: _Path
-    def __init__(self, fullname: str, path: _Path) -> None: ...
-    def get_data(self, path: _Path) -> bytes: ...
-    def get_filename(self, name: str | None = ...) -> _Path: ...
+    path: str
```

Comment:
This class has no implementation, but from `_bootstrap_external.FileLoader` it seems clear the path is not intended to be bytes: https://github.com/python/cpython/blob/2cfcaf5af602b297fc90086de4d8ac980c7891e2/Lib/importlib/_bootstrap_external.py#L1195